content/en/blog/_posts/2021-07-12-Kubernetes-Release-Cadence/index.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: blog
+title: "Kuberentes Release Cadence Change: Here’s What You Need To Know"
+date: 2021-07-01
```

Comment:
This date is a bit in the past, Do we know a future date that this blog post will be out? I do not want to put pressure on you, I want to reference this blog post in the Major themes for the 1.22 release notes.

Reply:
https://kubernetes.io/docs/contribute/new-content/blogs-case-studies/ has some more context on this (but could be more clear). The blog team can pick a date and update it to be in the future, so long as [maintainer pushes](https://kubernetes.io/docs/contribute/review/for-approvers/#commit-into-another-person-s-pr) to the source fork are permitted.

Reply:
The first challenge here is finding folks to do the first-pass review; once that's happened, I think it'll be easier to get a date set.